src/content/developers/docs/nodes-and-clients/nodes-as-a-service/index.md

```diff
@@ -11,12 +11,18 @@ preMergeBanner: true
 
 Running your own [Ethereum node](/developers/docs/nodes-and-clients/#what-are-nodes-and-clients) can be challenging, especially when getting started or while scaling fast. There are a [number of services](#popular-node-services) that run optimized node infrastructures for you, so you can focus on developing your application or product instead. We'll explain how node services work, the pros and cons for using them and list providers if you are interested in getting started.
 
-**Note: it has been common for users to run consensus clients locally and use a node-as-a-service provider in place of a local execution client. This will not be possible at The Merge - users will be required to run BOTH clients locally. Users relying on node-as-a-service providers for their consensus clients will not merge correctly and will not be able to follow the Ethereum chain. Now is the time to install a local execution client!**
-
 ## Prerequisites {#prerequisites}
 
 If you don't already have an understanding of what nodes and clients are, check out [Nodes and clients](/developers/docs/nodes-and-clients/).
 
+## Stakers
+
+Solo stakers are required torun their own infrastructure rather than relying on third party providers. This means running an execution client coupled to a consensus client. Before [The Merge](/upgrades/merge) it was possible to run a consensus client only and use a centralized provider for execution data. This is no longer possible - a solo staker must run both clients. There are services available to ease this process.
```

Comment:
```suggestion
Solo stakers must run their own infrastructure rather than relying on third-party providers. This means running an execution client coupled with a consensus client. Before [The Merge](/upgrades/merge), it was possible to run a consensus client only and use a centralized provider for execution data; this is no longer possible - a solo staker must run both clients. However, there are services available to ease this process.
```